system/kube-monitoring/charts/prometheus-collector/templates/_prometheus.yaml.tpl

```diff
@@ -312,6 +312,29 @@ scrape_configs:
     replacement: {{ $region }}
 {{ end }}
 
+{{ range $region := .Values.global.regions }}
+- job_name: 'blackbox-tcp-{{ $region }}'
+  metrics_path: /probe
+  scrape_interval: 15s
+  params:
+    module: [tcp_connect]
+  static_configs:
+    - targets:
+      {{- range $.Values.blackbox_exporter.tcp_probe_targets }}
```

Comment:
this assumes there's a `blackbox_exporter.tcp_probe_targets` value. surround the whole job with with`{{ if blackbox_exporter.tcp_probe_targets }} ... {{ end }}`,